.babelrc

```diff
@@ -6,7 +6,6 @@
           "env",
           {
             "targets": {
-              "browsers": ["last 1 versions"],
```

Comment:
Why are we removing this? Isn't this `.babelrc` important for supporting using the sdk in a browser?

Reply:
Building, the rollup target, further down includes the last two versions. This might only affect testing.